internal/pkg/cli/app_init.go

```diff
@@ -226,6 +233,19 @@ func (o *initAppOpts) validateAppName(name string) error {
 	return nil
 }
 
+func (o *initAppOpts) isDomainOwned() error {
+	err := o.domainInfoGetter.IsDomainOwned(o.domainName)
+	if err == nil {
+		return nil
+	}
+	var errDomainNotFound *route53.ErrDomainNotFound
+	if errors.As(err, &errDomainNotFound) {
+		o.isDomainNotFoundInAccount = true
+		return nil
```

Comment:
I wonder if we should error out and log out here instead of letting it go, since the user wants to create an app with domain but we in fact create an app without domain for them

Reply:
Yeah, I agree w boss. We should err here and log the recommended action here 🙏